CMakeLists.txt

```diff
@@ -611,7 +686,6 @@ list(APPEND JIT_TEST_SRCS
   ${NVFUSER_ROOT}/tests/cpp/test_statement_guard.cpp
   ${NVFUSER_ROOT}/tests/cpp/test_swizzle.cpp
   ${NVFUSER_ROOT}/tests/cpp/test_tensor_factories.cpp
-  ${NVFUSER_ROOT}/tests/cpp/test_tmem.cpp
```

Comment:
is this intended?

Reply:
🙇🏼 This is a copy from my out-of-date branch.

Reply:
It's not your fault, our automatic test/CI unfortunately won't catch it when a set of test is accidentally removed from build. :cry: 

I wonder if it's worth having a BC cpp test run in CI. Tagging @xwang233 for follow up in the future.

Reply:
Sorry I probably missed this thread. What kind of BC cpp test would you like to see? Does codegen diff help here if one of the test suite is missing? 🤔